contracts/JBTokenStore.sol

```diff
@@ -176,15 +183,21 @@ contract JBTokenStore is JBControllerUtility, JBOperatable, IJBTokenStore {
     // Can't set to the zero address.
     if (_token == IJBToken(address(0))) revert EMPTY_TOKEN();
 
-    // Can't set token if already set.
+    // Can't set token if the project is already associated with another token.
     if (tokenOf[_projectId] != IJBToken(address(0))) revert ALREADY_SET();
 
+    // Can't set token if its already associated with another project.
+    if (projectIdOf[_token] != 0) revert ALREADY_SET();
```

Comment:
I think this might make the current `IJBToken` obsolete, since a single token contract can no longer be used for multiple projects. Should we all the `_projectId` arguments entirely?

Reply:
you might be right. ill look into it too

Reply:
done in #32